patches/tModLoader/Terraria/ModLoader/Default/ModAccessorySlotPlayer.cs

```diff
@@ -86,6 +126,26 @@ public override void LoadData(TagCompound tag)
 		}
 	}
 
+	public override void OnEnterWorld()
+	{
+		// Need to enter world with disabled slots.
+		DetectConflictsWithSharedSlots();
+
+		if (extraItems.Count == 0)
+			return;
+
+		foreach (var items in extraItems) {
+			if(!items.Accessory.IsAir)
```

Comment:
Might be worth running an `(if|for|foreach)\(` to find missing spaces periodically